css-two-column-layout/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="column-full"">
+      <div class="row">
+        <h1>SCALE, SPEED, AGILITY</h1>
+      </div>
+      <div class="column-full">
+        <p class="header">We've dedicated thousands of man hours to making
+          healthcare institutions in the Helium<br> run 10x more
+          efficiently and deliver an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row-body">
+      <div class="column-one-half">
+        <div class="column-image">
+          <img src="images/software.svg">
+        </div>
+        <div class="column-text">
+          <h3 class="software">INTUITIVE, <br>SMART SOFTWARE</h3>
+          <p>We build products that transform how
+            healthcare facilities run across Africa
+            - making them radically more efficient
+            operationally and financially, a complete
+            turnaround.
+          </p>
+        </div>
+      </div>
+      <div class="column-one-half">
+        <div class="column-image">
+          <img src="images/practice.svg">
+        </div>
+        <div class="column-text">
+          <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+          <p>Our all-in-one electronic health records
+            (EHR), hospital management (HMIS) &amp;
+            billing system helps streamline your
+            facility, reduce waste and improve
+            quality of care.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row-body">
+      <div class="column-one-half">
+        <div class="column-image">
+          <img src="images/circumstance.svg">
+        </div>
+        <div class="column-text">
+          <h3>FOR ANY CIRCUMSTANCE</h3>
+          <p>Our intuitive system is built around the
+            peculiarities of your facility; we support
+            offline deployments, sync to the cloud
+            and run on any device 24/7.
+          </p>
+        </div>
+      </div>
+      <div class="column-one-half">
+        <div class="column-image">
+          <img src="images/data.svg">
+        </div>
+        <div class="column-text">
+          <h3>DATA THAT TRANSFORMS</h3>
+          <p>We understand your facility better than
+            you do; we provide visually-appealing
+            real-time analytics, reports and trends
+            at your finger tips.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full">
+        <form>
+          <button value="LEARN MORE">LEARN MORE</button>
+        </form>
```

Comment:
For this button, its purpose is more for CTA (call-to-action). And since there is no data that user is submitting in this case, we won't need `form`. So let's remove that.

`button` will do good for this exercise, but just FYI using `a` tag is more fitted in this case since "learn more" usually links to some other pages. You are welcome to change it to `a` if you want, but not required for this exercise.